src/Command/BakeMigrationCommand.php

```diff
@@ -120,7 +120,7 @@ public function templateData(Arguments $arguments): array
                 'primaryKey' => $primaryKey,
             ],
             'name' => $className,
-            'backend' => Configure::read('Migrations.backend', 'builtin'),
+            'backend' => 'builtin',
```

Comment:
We should be able to remove this and the template conditions. I can do another pass to get this stuff.